global_test.go

```diff
@@ -92,6 +92,18 @@ func TestGlobalsConcurrentUse(t *testing.T) {
 	wg.Wait()
 }
 
+func TestNewStdLog(t *testing.T) {
+	withLogger(t, DebugLevel, nil, func(l *Logger, logs *observer.ObservedLogs) {
+		std := NewStdLog(l)
+		std.Print("redirected")
+
+		assert.Equal(t, []observer.LoggedEntry{{
+			Entry:   zapcore.Entry{Message: "redirected"},
+			Context: []zapcore.Field{},
+		}}, logs.AllUntimed(), "Unexpected global log output.")
```

Comment:
nit: the output here isn't global. Please adjust the error message to clarify :)